### PR TITLE
Reset error banners when there is no submission errors

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#13

### CODE REVIEW

File: client/src/app/submission/submission.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  });
+
+  it('should be created', () => {
+    expect(submissionComponent).toBeTruthy();
+  });
+
+  describe('displaySubmissionErrors', () => {
+    it('clears error banners when there is no submission error', () => {
+      // given
+      let submissionEnvelope: SubmissionEnvelope;
```
Comment (ke4) [inline client/src/app/submission/submission.component.spec.ts:37]: Shall we merge line 37 and 38?

File: client/src/app/submission/submission.component.ts [hunk 2]
```diff
@@ -141,9 +141,9 @@ export class SubmissionComponent implements OnInit, OnDestroy {
     }
   }
 
-  private displaySubmissionErrors(submissionEnvelope: SubmissionEnvelope) {
+  public displaySubmissionErrors(submissionEnvelope: SubmissionEnvelope) {
```
Comment (ke4) [inline client/src/app/submission/submission.component.ts:144]: I am not happy about this as it should stay a private method.

**Review by ke4 — APPROVED** [review]
Added some comments, but approved it

**Review by ke4 — APPROVED** [review]

**Comment by aaclan-ebi** [comment]
We only have @ke4 available today. Merging this before i go on holidays.

**Pull request merged**